pom.xml

```diff
@@ -300,916 +301,321 @@
 
     <dependencyManagement>
         <dependencies>
+            <!-- geomesa modules -->
```

Comment:
I re-organized the pom to group things together better, but the imports did not change except for the addition of jedis